KIPs/kip-81.md

```diff
@@ -0,0 +1,122 @@
+A voter casts votes on the proposal during the voting period. A voter can vote for yes, no, or abstain with all the votes assigned to the voter. The list of voters for each proposal is determined at the proposal submission. The number of votes is determined by the amount of staked KLAYs at the start of the voting period.
+An executor triggers the execution of attached transactions in passed proposals. The execution is subdivided into two steps. First, the executor queues the transactions. After a set amount of delay, the executor can send the transactions. Execution delay here gives enough time for the community to recognize the upcoming change and perform a final check about the transaction.
+A proposal is in one of the following states.
+- Pending: Proposer submitted the proposal but voting has not started. The pending state lasts for votingDelay, or 7 days. This state is also called the notice period.
+- Active: Ongoing voting. Voters can cast their votes. The active state lasts for votingPeriod, or 7 days. This state is also called the voting period.
+- Canceled: Proposer has canceled the proposal before execution. Any further interaction is prohibited.
+- Passed: Voting finalized and quorum reached. Executor shall queue the actions before a set timeout of queueTimeout, or 7 days. If the proposal contains no actions, any further interaction is prohibited. Otherwise, the proposal can proceed to Queued and Executed states.
+- Failed: Voting finalized but has not reached quorum. Any further interaction is prohibited.
+- Queued: An executor has queued the actions. An executor cannot trigger the actions while a certain duration of execDelay, or 2 days.
+- Executed: An executor has triggered the actions after the execution delay. The executor shall trigger the actions before a set timeout of execTimeout, or 7 days. The execTimeout starts at the end of execDelay.
+- Expired: Executor has not queued or executed within respective timeouts.
+
+
+![proposal state diagram](../assets/kip-81/proposal_state_diagram.png)
+
+For a proposal to pass, two quorum conditions must be satisfied. The first condition is the turnout. Either the number of voters exceeds 1/3 of all voters, or the sum of casted votes exceeds 1/3 of all eligible votes. The second condition is supported. The yes votes must exceed the half of casted votes.
+The amounts of votes are calculated from the amounts of stakes KLAY. GC members stake their KLAY by locking in a special smart contract called CnStakingContract. Each GC must deploy at least one CnStakingContract and deposit KLAY there. The official addresses of CnStakingContract are managed in another special smart contract called AddressBook.
+Each GC member can appoint a voter account that will cast votes on behalf of the KLAYs in its CnStakingContracts. The voter account is necessary because CnStakingContract itself cannot send transactions to cast votes.
```

Comment:
```suggestion
Each GC member can appoint a voter account that will cast votes on behalf of the KLAY in its CnStakingContracts. The voter account is necessary because CnStakingContract itself cannot send transactions to cast votes.
```